Dockerfile.updater-core

```diff
@@ -125,6 +125,8 @@ RUN apt-get update \
     git-lfs \
     bzr \
     mercurial \
+    # needed to sign commits
```

Comment:
Oh this works??!

I thought `# comments` in the middle of single-line bash scripts that are made multi-line via `\` typically broke things... But I guessdocker is smart enough to strip the comment out before passing the command through? TIL!

Reply:
Yeah it's a Docker thing!